api_products.md

```diff
@@ -214,8 +214,10 @@ Product attributes:
 |--------------------|----------|-----------|--------------------------------------------------------------------------------------------------------------------|
 | `code`             | `string` | ""        | Scannable code / barcode                                                                                           |
 | `template`         | `string` | "default" | Identifier of the [code template](#code-template) used                                                             |
-| `transmissionCode` | `string` | null      | In case the POS cannot handle the scannable code / barcode, it contains a POS friendly code                        |
+| `isPrimary`        | `bool`   | null      | If a product has a code marked as primary this code is always presented to a POS instead of the scanned one. A product is not allowed to have more than one primary code. Optional. |
+| `transmissionCode` | `string` | null      | In case the POS cannot handle the scannable code / barcode, it contains a POS friendly code. Should not be used in ombination with primary codes.                        |
```

Comment:
```suggestion
| `transmissionCode` | `string` | null      | In case the POS cannot handle the scannable code / barcode, it contains a POS friendly code. Should not be used in combination with primary codes.                        |
```